src/EntryPoints/REST/ApprovePageApi.php

```diff
@@ -75,10 +78,16 @@ public function newSuccessResponse( ?ApprovalState $state ): Response {
 			return $this->getResponseFactory()->createNoContent();
 		}
 
+		$language = $this->languageFactory->getLanguage( $this->getRequest()->getHeaderLine( 'Accept-Language' ) );
+		$formattedTimestamp = TimestampFormatter::formatTimestamp( $state->approvalTimestamp, $language );
+
 		return $this->getResponseFactory()->createJson( [
-			'approvalTimestamp' => $state->approvalTimestamp,
+			'approvalTimestamp' => $formattedTimestamp,
```

Comment:
Anti pattern: putting a bunch of low level details for one of the values into this function rather than extracting it.

Reply:
TBH I'm not sure why you are changing this specific value from int to formatted string, as per the comment in the JS file.